working-groups.md

```diff
@@ -0,0 +1,65 @@
+# Working Groups 
+
+### [Business Planning][biz-wg]
+
+WG for performing feasibility study, and business planning to identify start up costs, fundraising mechanisms, and risks.
+
+📅 Weekly calls  
+🌱 Cultivators:   
+👪 Members: [**@business-planning-wg**][t-biz]  
+✅ Tasks: GitHub board > [`wg:business-planning` label][l-biz]  
+
+### [Finance][fin-wg]
+
+WG for establishing accounting practices, including setting up a bank account, administering payroll and expenses, invoicing, and handling remittances.
+
+📅 Weekly calls  
+🌱 Cultivators:  
+👪 Members: [**@finance-wg**][t-fin]  
+✅ Tasks: GitHub board > [`wg:finance` label][l-fin]  
+
+### [Governance][gov-wg]
+
+WG for drafting by-laws, articles of incorporation, and planning for the first General Meeting (and subsequent structure).
+
+📅 Biweekly calls  
+🌱 Cultivators: dc  
+👪 Members: [**@governance-wg**][t-gov]  
+✅ Tasks: GitHub board > [`wg:governance` label][l-gov]  
+
+### [Infrastructure][inf-wg]
+
+WG for setting up our "virtual office", in particular the infrastructure and processes to collaborate.
+
+📅 Biweekly(ish) calls  
+🌱 Cultivators: patcon & elon. (next rotation after..?)  
+👪 Members: [**@infra-wg**][t-inf]  
+✅ Tasks: GitHub board > [`wg:infra` label][l-inf]  
+
+### [Operations][ops-wg]
+
+WG for stewarding our shared practices and policies for effective remote collaboration.
+
+📅 We host biweekly calls  
+🌱 Cultivators:   
+👪 Members: [**@ops-wg**][t-ops]  
+✅ Tasks: GitHub board > [`wg:ops` label][l-ops]  
+
+<!-- Links: Working Groups -->
+[biz-wg]: https://link.hypha.coop/biz-wg
+[fin-wg]: https://link.hypha.coop/fin-wg
+[gov-wg]: https://link.hypha.coop/gov-wg
+[ops-wg]: https://link.hypha.coop/ops-wg
+[inf-wg]: https://link.hypha.coop/inf-wg
+
+[l-biz]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:business-planning"
+[l-fin]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:finance"
+[l-gov]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:governance"
+[l-ops]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:ops"
```

Comment:
We discussed to not change GitHub labels and team names, but looking at this it seems `wg:ops` and `wg:infra` should be written out as:

- `wg:ops` -> `wg:operations`
- `wg:infra` -> `wg:infrastructure`

Can I make the change across the org?

Reply:
I brought this up in the Ops call yesterday `2020-03-11`, I think these could be expanded to be consistent...

That said, I think "infra" and "ops" are used so heavily in many domains consistently to mean that thing that I think we could get by without.

My take on an approach: least possible change for an elegant solution.

Reply:
> our guideline is to have written out names

just noting that i'm unaware of this guideline :)

Reply:
I brought up in ops call we have human-readable written out names for labels (and in other places where humans read things), I didn't want to lose that for three letter short codes like those that are present in the links below.

@benhylau noticed we had like four ways of referring to each group, and wants to make them more consistent, which I fully support:
- full names (e.g., finance, governance)
- hybrid abbrev (e.g., bizdev)
- standard abbrev (e.g., infra, ops)
- 3-letter (e.g., inf, gov, fin)

We haven't been using those the same way in the same places. With @benhylau 's edits to template we are much more consistent. I think using a mix of:
- full names (e.g., finance, governance) for things people read (tho "infra" and "ops)
- 3-letter (e.g., inf, gov, fin) for things like links

This works for me, but if Ben would like to see through **TOTAL CONSISTENCY** I'm fine with that.

Reply:
Okay I am going to change everything to suit this policy:

- Internal references -> "3-letter (e.g., inf, gov, fin) for things like links"
- Public and visible -> "full names (e.g., finance, governance)"